addon_service/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.2.7 on 2024-08-09 15:34
+# Generated by Django 4.2.7 on 2024-09-09 11:49
```

Comment:
Can we not merge into the initial migration anymore? Now that we are on staging, we should be just making normal forward migrations from now on.